app/models/imported_item.rb

```diff
@@ -17,25 +17,25 @@ def import(params)
       contacts_rows << row
     end
 
-    unique_contacts, not_unique_contacts = unique_records contacts_rows
+    unique_contacts, not_unique_contacts = unique_and_non_unique_records contacts_rows
```

Comment:
Why is this now unique_and_non_unique? Was it wrong before?

Not sure I ever understood the syntax on this line actually, what does it mean to have `x, y = z`

Reply:
is it assigning both unique_contacts and not_unique_contacts the same thing?

Reply:
ah just looked up think I understand now. So `unique_and_non_unique_records contacts_rows` returns `[unique_contacts, not_unique_contacts]` and then each get assigned accordingly. Makes sense I think

Reply:
renaming of method makes sense in that case :)